src/styles/vendor/vue-directive-tooltip.css

```diff
@@ -1,85 +1,107 @@
-.vue-tooltip {
-  line-height: 1.5;
+.tooltip {
+  display: block !important;
+  z-index: 10000;
   font-family: var(--sans);
+  font-size: 12px;
+}
+
+.tooltip .tooltip-inner {
   background: black;
-  box-sizing: border-box;
   color: white;
-  padding: 0.5rem 0.75rem;
   border-radius: 0.25rem;
-  z-index: var(--z-modal);
-  font-size: var(--sm);
-  text-align: center;
-}
-
-.vue-tooltip .vue-tooltip-content {
-  text-align: center;
+  padding: 0.5rem 0.75rem;
 }
 
-.vue-tooltip .tooltip-arrow {
-  content: "";
+.tooltip .tooltip-arrow {
   width: 0;
   height: 0;
   border-style: solid;
   position: absolute;
-  margin: 0.375rem;
+  margin: 5px;
+  border-color: black;
+  z-index: 1;
 }
 
-.vue-tooltip[x-placement^="top"] {
-  margin-bottom: 0.375rem;
+.tooltip[x-placement^="top"] {
+  margin-bottom: 5px;
 }
 
-.vue-tooltip[x-placement^="top"] .tooltip-arrow {
-  border-width: 0.375rem 0.375rem 0 0.375rem;
-  border-top-color: black;
-  border-bottom-color: transparent !important;
+.tooltip[x-placement^="top"] .tooltip-arrow {
+  border-width: 5px 5px 0 5px;
   border-left-color: transparent !important;
   border-right-color: transparent !important;
-  bottom: -0.375rem;
+  border-bottom-color: transparent !important;
+  bottom: -5px;
+  left: calc(50% - 5px);
   margin-top: 0;
   margin-bottom: 0;
 }
 
-.vue-tooltip[x-placement^="bottom"] {
-  margin-top: 0.375rem;
+.tooltip[x-placement^="bottom"] {
+  margin-top: 5px;
 }
 
-.vue-tooltip[x-placement^="bottom"] .tooltip-arrow {
-  border-width: 0 0.375rem 0.375rem 0.375rem;
-  border-bottom-color: black;
-  border-top-color: transparent !important;
+.tooltip[x-placement^="bottom"] .tooltip-arrow {
+  border-width: 0 5px 5px 5px;
   border-left-color: transparent !important;
   border-right-color: transparent !important;
-  top: -0.375rem;
+  border-top-color: transparent !important;
+  top: -5px;
```

Comment:
Negative numbers 🚨 🚨🚨

Reply:
in this case it is ok, right?

Reply:
I don't know 🤷 

@jbibla , why was it OK here using negative numbers?

Reply:
i didn't write these styles - it's a vendor stylesheet